griptapecli/core/skatepark.py

```diff
@@ -91,9 +93,10 @@ def create_structure_run(structure_id: str, run: StructureRun) -> StructureRun:
         stdout=subprocess.PIPE,
         env={
             **os.environ,
+            "GT_CLOUD_RUN_ID": run.run_id,
+            "GT_CLOUD_BASE_URL": str(request.base_url),
```

Comment:
Is the right ordering? Or should they be before `os.environ` so they can be overridden by os envs?

Reply:
we want customer to be able to override, so this'd be a gotcha, no?

Reply:
I'm playing this back in my head, let me know if this is comprehensive and accurate:
1. In Skatepark emulator, customer wants to override the base URL if they've selected a different URL or port. So having this up here and then overridden by os.environ makes sense.
2. In Cloud, customer never wants to override the base URL, right? Or are there scenarios where they might?
3. In all situations, what is the benefit to overriding the Run ID? Should that be explicit?

Reply:
I think rather than try to predict when a user may or may not want to override these environment variables it's better to just have a consistent behavior for overrides.

1. We provide some system level environment variables (`GT_CLOUD_RUN_ID` and `GT_CLOUD_STRUCTURE_ID`).
2. Then `os.environ` is layered on.
3. Then `structure.env` is layered on.
4. Then `run.env` is layered on.

If you want to get fancy and override these, use at your own risk. If they try to set values for resources they don't own, auth will block them anyways.

Reply:
That works for me. So what's the full set of changes we need to make to enshrine that?

Reply:
This is how it works with the current changes.